.github/workflows/ci.yml

```diff
@@ -75,18 +68,21 @@ jobs:
       - name: Install Python dependencies
         run: |
           python -m venv venv
-          . venv/bin/activate
-          pip install --upgrade pip wheel
-          pip install -r requirements.txt
+          source venv/bin/activate
+
+          pip install --upgrade pip==25.0.1 pip-tools
+
+          pip-compile --output-file=requirements_dev.txt requirements_dev.in
```

Comment:
So requirements_dev.txt are for local development only so I think you just want requirements.txt, here